spec/models/concerns/digital_object_concerns/attribute_assignment/rights_spec.rb

```diff
@@ -44,7 +44,7 @@
     end
 
     it 'works as expected on dynamic_field_data instance variable' do
-      digital_object_with_sample_data.set_rights('rights' => rights)
+      digital_object_with_sample_data.assign_rights('rights' => rights)
```

Comment:
does assign_rights have the same optional merge behavior as the dynamic field data?

Reply:
no it doesn't, afaik

Reply:
Should we ticket that? Seems like a potential gotcha down the line.

Reply:
are you thinking that merge is probably going to be necessary when importing through a CSV? Because I don't see a use case for merging when the data is coming from the form.

Reply:
Yes - for the basic fields for example (if they want to bulk manage copyright status) - but also for API uniformity in two very similar methods/operations.